openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1134,6 +1138,160 @@ def parse_smirnoff_from_source(self, source):
         msg += f"Parsing failed with the following error:\n{exception_msg}\n"
         raise IOError(msg)
 
+    def _terms(self):
+        """
+        Generate a list of active term keys. These keys are can be used to find the
+        specific value. The order the keys will be the same, and will only differ
+        if the order is manually changed, or the list of active terms is changed.
+
+        This should be public, but we make it private to hide from docs
```

Comment:
Since the request was to make this entire PR only introduce underscore methods, I added this line to methods where I think should be eventually public (i.e. I plan to use them downstream) , as opposed to underscore methods in the PR that are _actually_ underscore methods.

Reply:
Sounds like a good plan to me!